libs/datasets/taglib.py

```diff
@@ -314,15 +314,18 @@ def content(self) -> str:
 @dataclass(frozen=True)
 class Derived(TagInTimeseries):
     TAG_TYPE = TagType.DERIVED
+    # Name of the function which added this derived tag.
+    f: str
```

Comment:
It would be nice to make this name a bit more longer - I'm guessing its to save space in the csv. but since you're manually serializing/deserializing probably can make this a bit longer

Reply:
Done in 2b58e223